relayer/src/config.rs

```diff
@@ -73,6 +73,10 @@ pub mod default {
     pub fn connection_delay() -> Duration {
         ZERO_DURATION
     }
+
+    pub fn wallet_metric_scale() -> u32 {
+        0
```

Comment:
Default = 6?

Reply:
We ended up deciding to use 0 as a default. I will update the PR description